io.jamulus.Jamulus.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: io.jamulus.Jamulus
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: Jamulus
+rename-icon: jamulus
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--share=network'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--socket=system-bus'
```

Comment:
What specifically does this need to talk to on the system bus? You should really log this with `flatpak run --log-system-bus` and then allowlist the specific calls (or check the source code to make sure you've got everything).

Reply:
I am looking into this. Thanks for the pointer!

Reply:
Maybe @ranfdev can give some insight, as he as the one who put together the first version.

Reply:
From [this comment](https://github.com/flatpak/flatpak/issues/1509#issuecomment-751475728) this is needed for the program to set real-time priority.

Reply:
@hfiguiere what's the context for that permission? On my system at least I get:
> [W][000105890.989482][module-rtkit.c:502 idle_func()] could not make thread realtime: Input/output error

With the current settings (but that's likely to be host system dependent I assume)

Reply:
If you get that error that mean the permission is needed. Otherwise you get an dbus error.

Reply:
Well, I'm getting that with the permission being granted. But what actual endpoint is it trying to talk to? This should be enumerable right rather than just saying "you can send anything to anything in the system"?

Reply:
I went ahead with the merge as I couldn't get it to work without `system-bus` while I couldn't get anything useful from `--log-system-bus` either.

Reply:
FWIW I have never been able to isolate which one is required.